.github/workflows/doctests.yml

```diff
@@ -20,7 +20,7 @@ env:
 
 jobs:
   run_doctests:
-    runs-on: [self-hosted, doc-tests-gpu]
+    runs-on: [single-gpu, nvidia-gpu, t4, past-ci]
```

Comment:
Maybe we can keep the `t4` tag on the runner enrollment to identity which hardware it is running and, in the future, if you end up having multiple flavoursl ike we'll have for amd `[mi210, mi250, mi300]` you can easily target them (or not).

See how we would like to approach it for AMD gpu flavours: https://github.com/huggingface/transformers/pull/26346/files#diff-0517e7d6ca0b03d887391fd62602e2891e13f279d32d464e3f73c6e864b8152fR48

Within the workflow I agree, at the moment, it might not bring any additional constraints.

Reply:
Technically  no need `t4`, it's there just to keep more info. explicitly.